.travis.yml

```diff
@@ -5,5 +5,5 @@ python:
 install:
     - pip install -r requirements.txt -e .
     - mkdir ~/.sparkmagic
-    - cp remotespark/default_config.json ~/.sparkmagic/config.json
+    - cp remotespark/example_config.json ~/.sparkmagic/config.json
```

Comment:
It was never actually "needed".

It's useful as a template for making your own config file, which we need to have some documentation on somewhere.  I vote we keep it.